mmdet3d/models/utils/edge_indices.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+def get_edge_indices(img_metas,
+                     step=1,
+                     pad_mode='default',
+                     dtype=torch.float32,
+                     device='cpu'):
+    """Function to filter the objects label outside the image.
+
+    Args:
+        img_metas (list[dict]): Meta information of each image, e.g.,
+            image size, scaling factor, etc.
+        step (int): Step size used for generateing edge indices.
+            Default: 1.
+        pad_mode (str): Padding mode during data pipeline.
+            Default: 'default'.
+        dtype (torch.dtype): Dtype of edge indices tensor.
+            Default: torch.float32.
+        device (str): Device of edge indices tensor. Default: 'cpu'.
+
+    Returns:
+        tuple:
```

Comment:
Should be a list?
No need for the variable name here, just revise it as "list[Tensor]: xxxxx"